### DISCUSSION TITLE
VIM activated but not working

### DISCUSSION THREAD

**Discussion by K1ubra1t** [Mr-MIBonk/M.I.B._More-Incredible-Bash#68] [Q&A] [answered]
Hi,

I really appreciate the work you are doing here. Thank you !

There was a discussion before with the same issue - VIM activated but not working. It is the same for me. The firmware is MU1177-MHI2Q_ER_AUG22_P4212 and the lo is nearly the same - 
![Log](https://user-images.githubusercontent.com/119583101/205017957-49bf5de7-c72b-4ada-b60b-a4ad344123de.png)

..it says "False checksum"

Is there any solution for that yet?
Thank you

**Comment by harman-f** [comment, marked as answer]
Must be some issue with that specific FW.
Same code is working on all other MU without an issue.

**Reply by K1ubra1t** [reply]
Thank you for the reply.
Is it possible to update the firmware?

**Reply by Piston73** [reply]
Did you do the update?
